src/index.html

```diff
@@ -1,14 +1,100 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <a class="logo" href="moyo__logo">
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+          class="image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
```

Comment:
If you have two attributes and they fit into one line, you do not need like break. Fix in all cases
```suggestion
            <a class="nav__link is-active" href="#apple">
```

Reply:
![image](https://user-images.githubusercontent.com/94186386/185781033-0677080c-bfcc-45f4-b1a7-4f0a9affcbc0.png) Я не переносил может это настройки вс код )?